### ISSUE TITLE
[AUDIT_READY] PR862 exact-head EDRSR real-execution qualification

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1072]
INDEPENDENT READ-ONLY AUDIT REQUEST

TARGET_PR: #862
TARGET_HEAD: 74e39eee6b164100ee189a3a07346abe45d88f1f
PRODUCT_OWNER_COMPLETED: #1070
PRODUCT_BRANCH: d03/overthelex-edrsr-train-candidate-20260907
CLEAN_SHARED_CI: 34537608369 = SUCCESS
REAL_EXECUTION_RUN: 34537415518 = SUCCESS
EXECUTION_A_JOB: 103072145274 = SUCCESS
EXECUTION_B_JOB: 103072145498 = SUCCESS
DETERMINISM_COMPARE_JOB: 103072308130 = SUCCESS
DURABLE_EVIDENCE: evidence/d03-overthelex-edrsr-real-execution-v1.json

AUDIT SCOPE — exact-head, independent, read-only:
1. Verify pinned train-only source identity and rights/provenance contract.
2. Verify validation/test payload was not accessed by execution path and no final-test leakage was introduced.
3. Verify only `facts` can enter candidate payload; no outcome/dispositive/judge metadata leakage.
4. Verify source SHA/bytes/schema/row-count checks fail closed.
5. Verify deterministic capped-prefix semantics are represented honestly: 827 retained / 5,993,276 normalized bytes; prefix disposition counts must not be misread as full-source row-level privacy/quality assessment.
6. Verify independent A/B identities and compare gate match durable evidence exactly.
7. Verify temporary workflow is absent on target head and shared CI is workflow-budget-clean.
8. Verify zero-credit boundary remains intact: no canonical capacity, training-authorized bytes, unique-loss positions, tokenizer fit, optimizer updates, model training or learned weights.
9. Identify any HIGH/CRITICAL blocker before integration; otherwise return exact-head PASS_FOR_INTEGRATION_MECHANICS_AND_REAL_EXECUTION.

EXPECTED REAL EXECUTION IDENTITIES:
source_sha256=e5706a2a35a0131224dc75eac8345296e11518688c4072dc996b25a55a5b7793
source_bytes=49826322
candidate_sha256=1d51f614f035d31555c52a552e2dc9c8806d600ce09627d836df863560407f13
report_file_sha256=d195f9cd07745bb1deab5dca5a52261e56f11531b1684b93434b86acf335be57
disposition_sha256=fe3204d4db4b9f2032fe386dc0150e8c27bb52d6e3a0b86f071e1f87ebcf1075
retained_records=827
retained_normalized_bytes=5993276

TRUTH BOUNDARY:
REAL_DATA_EXECUTED=true
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false

Do not mutate Product branch while auditing. Do not treat audit PASS as training authorization; downstream current canonical global dedup/decontamination/privacy/split/packing/unique-loss gates remain mandatory.

**Comment by Oleksii-debug** [comment]
INDEPENDENT AUDIT-B COMPLETE — exact target head `74e39eee6b164100ee189a3a07346abe45d88f1f` remained unchanged through verdict.

VERDICT: `PASS_FOR_INTEGRATION_MECHANICS_AND_REAL_EXECUTION`.

Verified independently against actual Actions metadata/logs and ancestry, not just producer JSON: execution run `34537415518` = SUCCESS at exact execution head `1790b11569a71bf89a1b90e3d853fe8151155abc`; execution A/B + compare all SUCCESS; exact train source SHA/bytes and deterministic candidate/report/disposition identities match the durable evidence; validation/test were not downloaded; materializer emits facts-only candidate content; prefix caveat is explicit; post-execution changes only removed the temporary workflow and added durable text-free evidence; final shared CI `34537608369` = SUCCESS on `74e39eee...`.

No HIGH/CRITICAL blocker found in requested scope. Detailed exact-head review is on PR #862 (review 5172910595).

This is NOT training/capacity authorization: canonical_capacity=0, training_authorized_bytes=0, authorized_unique_loss_positions=0, tokenizer_fit=false, optimizer/model training=false, learned weights=false, reserved-evaluation decontamination incomplete, final-test access=false, paid compute=false, foreign pretrained weights=false. Downstream canonical dedup/decontamination/privacy/quality/balance/split/packing/two-build/unique-loss gates remain mandatory.

**Comment by Oleksii-debug** [comment]
CONVERGENCE TERMINALIZATION — canonical independent auditor #1074 already completed this exact-head request.

TARGET_PR=#862
TARGET_HEAD=`74e39eee6b164100ee189a3a07346abe45d88f1f`
VERDICT=`PASS_FOR_INTEGRATION_MECHANICS_AND_REAL_EXECUTION`
FINAL_SHARED_CI=`34537608369=SUCCESS`
REAL_EXECUTION_RUN=`34537415518=SUCCESS`

The audit verified exact train-only source identity, A/B deterministic real execution, facts-only output, validation/test non-download, explicit capped-prefix semantics, workflow cleanup and zero-credit boundary. #862 remains open/mergeable at the audited head (currently draft), so no re-audit is warranted. Closing this handoff as satisfied and routing the qualified candidate to #1045.

This preserves training_authorized_bytes=0, authorized_unique_loss_positions=0, optimizer/model training=false, learned weights=false, final-test access=false, paid compute=false, foreign pretrained weights=false.
